scripts/bootstrap.sh

```diff
@@ -8,8 +8,8 @@ xcode-select --install
 
 echo "Checking for Homebrew..."
 if test ! $(which brew); then
-  echo "Installing homebrew..."
-  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+  echo "You must install homebrew, see https://brew.sh/"
```

Comment:
Now after brew install user must do : 

```
echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/"YOUR USER NAME"/.zprofile
eval $(/opt/homebrew/bin/brew shellenv)
```

I didn't want to waste time automating this so I took the homebrew installation out of the script.